src/components/Operatives/MobileWorkingWorkOrderDetails.js

```diff
@@ -62,82 +55,83 @@ const MobileWorkingWorkOrderDetails = ({
 
         {workOrder.plannerComments && (
           <>
-            <div className="lbh-heading-h5">Planner Comment</div>
+            <h5 className="lbh-heading-h5">Planner Comment</h5>
             <p className="govuk-body">{workOrder.plannerComments}</p>
           </>
         )}
+        <div className="work-order-information">
+          {cautionaryAlertsType && (
+            <GridRow className="align-grid-row">
+              <GridColumn width="one-half">
+                <a
+                  className="lbh-heading-h5 lbh-link"
+                  href="#"
+                  id="caut-alerts"
+                  onClick={cautContactURL}
+                >
+                  Caut. alerts
+                </a>
+              </GridColumn>
+              <GridColumn width="one-half" className="align-grid-column">
+                <div className="govuk-warning-text lbh-warning-text">
+                  <span
+                    className="govuk-warning-text__icon person-alert--icon"
+                    aria-hidden="true"
+                  >
+                    !
+                  </span>
+                  <strong className="govuk-warning-text__text person-alert--text">
+                    {cautionaryAlertsType}
+                  </strong>
+                </div>
+              </GridColumn>
+            </GridRow>
+          )}
 
-        {cautionaryAlertsType && (
-          <GridRow>
+          <GridRow className="align-grid-row">
```

Comment:
Best to replace this custom class with just a good old `govuk-!-margin-top-0`. It's normally best to use any utility classes when we want single declarations like this and they are available.